lib/simple_form/inputs/base.rb

```diff
@@ -189,9 +189,14 @@ def translate_from_namespace(namespace, default = '')
 
       def merge_wrapper_options(options, wrapper_options)
         if wrapper_options
-          options.merge(wrapper_options) do |_, oldval, newval|
-            if Array === oldval
-              oldval + Array(newval)
+          wrapper_options.merge(options) do |key, oldval, newval|
+            case key.to_s
+            when "class"
```

Comment:
I prefer to not, unless we have proof that this will improve significantly the performance of the application.